doc/introduction/templates.rst

```diff
@@ -110,6 +110,11 @@ neural network. Note arbitrary templates or operations can also be layered using
     :description: BasicEntanglerLayers
     :figure: ../_static/templates/layers/basic_entangler.png
 
+.. customgalleryitem::
+    :link: ../code/api/pennylane.templates.layers.ParticleConservingU1.html
+    :description: ParticleConservingU1
+    :figure: ../_static/templates/layers/particle_conserving_u1.png
+
```

Comment:
The picture is distorted, maybe one has to put a rectengular white background to increase the width?

Reply:
This looks OK now.